tests/post_training/conftest.py

```diff
@@ -145,9 +157,12 @@ def accept_pipeline_type(cls, pipeline_type: PipelineType) -> bool:
     @classmethod
     @TableColumn.assign_default_value
     def get_value(cls, info: Dict[PipelineType, RunInfo], target_pipeline_type: PipelineType) -> str:
-        if info[PipelineType.FP32].FPS > 1e-5:
-            return info[target_pipeline_type].FPS / info[PipelineType.FP32].FPS
-        return "inf"
+        if info[target_pipeline_type].fps is None or info[PipelineType.FP32].fps is None:
+            return NOT_AVAILABLE_MESSAGE
```

Comment:
Oh, finally we avoid negative metrics in reports, thanks a lot!!!!

Reply:
More, i will add xml table like in e2e in ci artifacts.